### ISSUE TITLE
Bug in get_percentile_arr function

### ISSUE THREAD

**Issue by jackaranda** [cerfacs-globc/icclim#74]
The get_percentile_arr function in calc_percentiles attempts to access the .mask attribute of the data array.  But the mask attribute does not always return a mask array (see masked array documentation).  Its more reliable to use the numpy.ma.getmaskarray(..) function.

**Comment by pagecp** [comment]
Yes this was fixed in other parts of icclim earlier, but not in that function... will also accept the merge request later today.
